processors/nitf21GeospatialConverters/decimalDegreeConverter.ts

```diff
@@ -18,12 +18,7 @@ export class DecimalDegreeConverter extends BaseProcessor {
                 const nitfIGEOLO = this.executionContext.getParameterValue('IGEOLO');
                 if (!nitfIGEOLO || nitfIGEOLO.length !== 60) {
                     errString = `${this.className} - Invalid IGEOLO: ${nitfIGEOLO}`;
-                    console.error(errString);
-                    return reject({
-                        httpStatus: 400,
-                        message: `${errString}`,
-                        successful: false,
-                    });
+                    return reject(this.handleError({message: errString}, `decimalDegreeConverter.fx`, 400));
```

Comment:
Should this use `${this.className}.fx` instead of the string?  (Again this goes back to the dynamic versus hardcoded values for error return messages)

Reply:
Good idea. Didn't think to use the className variable defined at the start of the class. Will change and commit.

Reply:
Change applied to all converters